platform-sdk/swirlds-state-api/src/main/java/com/swirlds/state/spi/MigrationContext.java

```diff
@@ -63,7 +63,8 @@ public interface MigrationContext {
      *
      * @return The {@link NetworkInfo} of the network at the time of migration.
      */
-    NetworkInfo networkInfo();
+    @Nullable
+    NetworkInfo genesisNetworkInfo();
```

Comment:
Again, isn't this also the network info used in reconnect (or other non-genesis) scenarios?

Reply:
The `networkInfo` is essentially set on `MigrationContextImpl` and is used in `AddressBookService` and `TokenService` only on `GenesisTrigger`